pkg/components/contour/component.go

```diff
@@ -28,7 +28,11 @@ import (
 	"github.com/kinvolk/lokomotive/pkg/util/walkers"
 )
 
-const name = "contour"
+const (
+	name                    = "contour"
+	serviceTypeNodePort     = "NodePort"
```

Comment:
Should we use https://godoc.org/k8s.io/api/core/v1#ServiceType instead?

Reply:
This entails adding a dependency, which seems more complex to me.
What is you rationale for using the constants from the k8s package?

Reply:
It's less duplication to me and it better reflects what the property represents. Our project depend on K8s anyway, so it's not adding a dependency to the project. But I don't insist on that.

Reply:
Right, it's adding a dependency to the package.